### ISSUE TITLE
The contrast between foreground and background colors must meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#1738]
Description  
---
The **code preview** (packages/storefront/src/components/CodeBlock.vue) color pallette for the components doesn't meet the WCAG color contrast guidelines. See detailed issue description on AXE Monitor: https://porsche-axemonitor.dequecloud.com/worldspace/organizationProject/issues/10?ruleId=color-contrast&status=open&severity=1&share=79aa45454cd0a2a10c1133c4deffa6a24ffdbe4a 

![image](https://user-images.githubusercontent.com/42934679/150985564-3b87e5d4-8932-4dc3-8365-ff3edd122510.png)
![image](https://user-images.githubusercontent.com/42934679/150985624-6cf3b774-b2d9-4d0c-9be5-8eaccb668e88.png)

Out of Scope
---
No replacement of code preview rendering - just colors

To Reproduce
---
1. Go to any component in Storefront
2. Check page with AXE DevTools
3. See color contrast issues 

Expected behavior
---
Color contrasts should meet WCAG Success Criterion 1.4.3 Contrast (Minimum)
Use https://github.com/PrismJS/prism-themes/blob/master/themes/prism-one-dark.css and https://github.com/PrismJS/prism-themes/blob/master/themes/prism-one-light.css

Test Environment
---
Platform/Browser/Version: all
